python/ray/serve/drivers.py

```diff
@@ -0,0 +1,21 @@
+from typing import Callable, Optional, Union
+
+import starlette
+
+from ray import serve
+from ray.serve.api import DAGHandle
+
+
+@serve.deployment
```

Comment:
should this be plain class and only turned into deployment where it's needed ?

Reply:
Resolved by removing these changes from this PR.